### ISSUE TITLE
Library status

### ISSUE THREAD

**Issue by lloydjatkinson** [richardszalay/mockhttp#98]
Hi, thanks for creating this library! 😄 It is more straightforward than the other manual approaches out there. I noticed since the last time I used it there's not been any commits (apart from one a couple of days ago) or new published versions.

Is the library looking for funding? I'm happy to fund it for a while to ensure it still gets maintained.

**Comment by richardszalay** [comment]
Hi @lloydjatkinson,

tl;dr I have less time than I used to, but I don't see the library as stagnating - it does exactly what it's meant to do and I still use it daily. Future .NET versions may drive changes, but beyond that I don't see it requiring constant updates.

MockHttp has been in development for 8 years - in that time, it's has its rough edges smoothed out and evolved features that people need. I've made heavy use of it in every professional project I've worked on since it's inception, and I know I'm far from the only one.

Beyond that, I do plan on reworking the build system (and dropping legacy .NET support in the process) and possibly adding some JSON-friendly `With` overloads but honestly those nice-to-have's since almost everyone has gotten by without them.

There are a few merged PRs from the community that I haven't released yet, and I do plan on getting around to that but at least one of them has breaking changes (from an API extensibility perspective) so it requires some more attention/thought before I move forward with a release.

Be assured that if a future release .NET brought with it compatibility issues (or major opportunities for improvement) I'd be all over it.

Appreciate the offer for funding, but it's not necessary. Thanks anyway!

Rich
